projects/ancestor/ancestor.py

```diff
@@ -0,0 +1,58 @@
+class Stack():
+    def __init__(self):
+        self.stack = []
+    def push(self, value):
+        self.stack.append(value)
+    def pop(self):
+        if self.size() > 0:
+            return self.stack.pop()
+        else:
+            return None
+    def size(self):
+        return len(self.stack)
+
+def earliest_ancestor(ancestors, person):
```

Comment:
Nice job. My only recommendation would be to pull out the DFT function from this and maybe utilize the Graph class you used in the other assignment to make earliest ancestor a little more readable.